CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.2.0-rc.2 - 2019-01-29
+
+### Added 
+- New `secrets_manager_vault_max_token_ttl` metric, so a user could alert based on this and `secrets_manager_token_ttl`
+  
+### Fixed
+- Deprecates `secrets_manager_vault_token_expired` metric as it was quite confusing since it's not really possible for `secrets-manager` to actually not it's expired, just "close to expire".
```

Comment:
```suggestion
- Deprecates `secrets_manager_vault_token_expired` metric as it was quite confusing since it's not really possible for `secrets-manager` to know when it's expired, just when it's "close to expire".
```

Reply:
ahh I wanted to type => `to actually know` it's just a typo